### DISCUSSION TITLE
[Feature] Validate before running `All` job

### DISCUSSION THREAD

**Discussion by PratikMahajan** [immich-app/immich#2501] [Feature Request]
### The feature

Clicking `All` instead of `Missing` on `Recognize Faces` job resets all face bindings and we need to run the job again to generate faces. 
There should be some kind of validation like `Do you really want to run ___ job on all resources?`, so that mistakenly clicking `All` does not reset already existing data. 
Maybe, add this to all `All` Jobs ?

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by jrasm91** [comment]
Implemented in #2496. No other jobs delete anything, so it is pretty unique to this one.
